CHANGES.rst

```diff
@@ -1,3 +1,614 @@
+4.3 (unreleased)
```

Comment:
Do you have to insert a real date here now, or is that for later?

Reply:
Aside from this question, https://astropy--11856.org.readthedocs.build/en/11856/changelog.html#unreleased seems to render fine, though I did not actually check to see if every entry is where they are supposed to be; I hope one of those release scripts have already taken care of that.

Reply:
Also, could you squash all commits together at this point? It's starting to get very incremental.

Reply:
I couldn't find a way to flatten it down to one that didn't end up removing some context for interpreting this PR thread, but I got it down to 5 at least, removing all the repetitve ones.

Reply:
Oh, and re: the "real date", that goes *after* the release, since you never know when there will be a delay that pushes the date forward...